roofit/roofitcore/src/RooProdPdf.cxx

```diff
@@ -687,7 +687,7 @@ void RooProdPdf::factorizeProduct(const RooArgSet& normSet, const RooArgSet& int
 	termNormDeps = new RooArgSet("termNormDeps");
 	termAllDeps = new RooArgSet("termAllDeps");
 	termIntDeps = new RooArgSet("termIntDeps");
-	termIntNoNormDeps = new RooArgSet("termIntNoNormDeps");
+	termIntNoNormDeps = &depIntNoNormList.emplace_back("termIntNoNormDeps");
```

Comment:
That's c++17 if I'm not mistaken? In 11, it returns `void`?

Reply:
You are right, I had to change that in a later commit to make the builds pass